docs/content/standards/coin.mdx

```diff
@@ -217,33 +217,18 @@ Use the following functions to get the values for the respective fields on the m
 
 Use the `coin::supply` function to get the current supply of a given coin.
 
-### Check for address on deny list
-
-Use the `coin::deny_list_contains` function to check if an address exists on the deny list for your coin. The signature of the function is:
-
-```move
-public fun deny_list_contains<T>(
-  freezer: &deny_list::DenyList,
-  addr: address
-): bool
-```
-
-The function returns `true` if the address is found on the coin's list, otherwise it returns `false`.
-
```

Comment:
Did we remove this feature, or document `deny_list_v2_contains_current_epoch` and `deny_list_v2_contains_next_epoch`?

Reply:
It's mostly for internal (per TN) use so removed.